### PR TITLE
Awesome Books website: Milestone-1

### PR DESCRIPTION
**Pull request by AlierPM**
* In this project milestone,  we built a basic website that allows users to add/remove books from a list. 
* We achieved this by using JavaScript objects and arrays. 
* We modified the DOM and added basic events dynamically.
*  [The Live Demo is reachable with this link](https://walterokumu.github.io/awesome-books/)

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    removeButton.id = index;
+    removeButton.innerText = 'Remove';
+
+    bookDiv.appendChild(h4);
+    bookDiv.appendChild(p);
+    bookDiv.appendChild(removeButton);
+    bookDiv.appendChild(hr);
+  });
+}
+
```
Comment (julie-ify) [inline scripts/index.js]: - [x] Your app does not work as expected. When I add a book and refreshed the page, the local storage is cleared and so nothing is displayed on the page. Kindly note that changes are expected to persist on the local storage and available books should be present on reload.

  A book is created | Browser is reloaded
  :---:|:---:
  hm-1 | hm-2

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @AlierPM @WalterOkumu,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/index.js [hunk 2]
```diff
@@ -56,6 +56,14 @@ function populateBook() {
   });
 }
 
+function loadFromStorage() {
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    bookList = JSON.parse(localStorage.getItem('bookList'));
+    populateBook();
+  }
+}
```
Comment (Kingstalux) [inline scripts/index.js]: - [x] Good job so far :+1: but right now I cannot even add a book due to this console error and as a result, I cannot verify if the local storage works. Kindly fix this issue and make sure that your app works as expected :100: 
- Console error:
Screenshot from 2022-08-09 08-02-29

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights :clap: 
:ballot_box_with_check: Good commit history :+1: 
:ballot_box_with_check: Linters are passing :ok_hand: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @WalterOkumu, @AlierPM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**